src/sage/rings/multi_power_series_ring.py

```diff
@@ -1019,6 +1020,19 @@ def ngens(self):
         """
         return self._ngens
 
+    def gens(self):
+        """
+        Return the generators of this ring.
+
+        EXAMPLES::
+
+            sage: M = PowerSeriesRing(ZZ, 3, 'v')
+            sage: M.gens()
+            (v0, v1, v2)
+        """
+
```

Comment:
This blank line is unconventional, no?

Reply:
oui, c'est vrai, merci. J'ai corrigé ça.